tk.turtlico.Turtlico.json

```diff
@@ -0,0 +1,166 @@
+{
+    "app-id": "tk.turtlico.Turtlico",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "turtlico",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=all",
+        "--env=PYTHONPATH=/usr/lib/python3.7/site-packages",
+        "--env=GPIOZERO_PIN_FACTORY=native"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/debug",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name" : "gtksourceview",
+            "buildsystem": "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/gtksourceview.git",
+                    "branch" : "4.4.0",
+                    "commit": "6cbb07d5454ccb46ee053cc458391289cc2fb630"
+                }
+            ]
+        },
+        {
+            "name" : "tcl",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://prdownloads.sourceforge.net/tcl/tcl8.6.10-src.tar.gz",
+                    "sha256" : "5196dbf6638e3df8d5c87b5815c8c2b758496eb6f0e41446596c9a4e638d87ed"
+                }
+            ],
+            "subdir" : "unix",
+            "config-opts": [
+                "--enable-threads"
+            ],
+            "post-install" : [
+                "chmod 755 /app/lib/libtcl8.6.so"
+            ]
+        },
+        {
+            "name" : "tk",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://prdownloads.sourceforge.net/tcl/tk8.6.10-src.tar.gz",
+                    "sha256" : "63df418a859d0a463347f95ded5cd88a3dd3aaa1ceecaeee362194bc30f3e386"
+                }
+            ],
+            "subdir" : "unix",
+            "config-opts": [
+                "--with-x",
+                "--enable-threads"
+            ],
+            "post-install" : [
+                "chmod 755 /app/lib/libtk8.6.so"
+            ]
+        },
+        {
+            "name" : "python3",
```

Comment:
Is python3 rebuilt here just for tkinter? if it's the case, you can just install it from pip sources without re-building  pyton3.

Reply:
Never mind.